sgkit/io/vcf/vcf_reader.py

```diff
@@ -126,13 +186,31 @@ def vcf_to_zarr_sequential(
                 call_genotype[i, ..., n:] = fill
                 call_genotype_phased[i] = gt[..., -1]
 
+                for info_field, arr in info_field_arrays.items():
+                    if info_field in [k for k, _ in variant.INFO]:
+                        if isinstance(variant.INFO[info_field], tuple):
+                            # for the case Number='.', just use the first value
```

Comment:
It might be better to also throw an error for the '.' case as this is commonly used indicate that the field is variable in length, see this old [htslib issue](https://github.com/samtools/htslib/issues/664)

Reply:
Agreed - let's not support this until it's done properly.